app/controllers/comments_controller.rb

```diff
@@ -51,4 +51,17 @@ def set_comment
   def comment_params
     params.require(:comment).permit(:content)
   end
+
+  def create_comment
+    @create_comment ||= ::Comments::Create.call(current_user: current_user, task: @task, comment_params: comment_params)
+  end
+
+  def update_comment
+    @update_comment ||= ::Comments::Create.call(current_user: current_user, task: @task,
+                                                comment_params: comment_params, comment: @comment)
+  end
```

Comment:
Предложение по code convention, однако твой вариант тоже хорош
```suggestion
  def update_comment
    @update_comment ||= ::Comments::Create.call(current_user: current_user,
      task: @task, comment_params: comment_params, comment: @comment)
  end
```